### DISCUSSION TITLE
links/

### DISCUSSION THREAD

**Discussion by giscus** [debuginn/blog#148] [General]
# links/

申请友联 申请友情链接的话请提交 issue 来申请。
1 2 3 4 - nickname: Debug客栈 avatar: https://image.debuginn.cn/202302182238057.JPG url: https://debuginn.cn description: 做一个爱分享的程序猿 友情提示 如果您想交换链接，请按上述格式留言。(仅限个人非商业性博客网站)； 网站出现故障、停止维护和不当内容可能会被取消链接！ 那些不尊重他人劳动成果、无源转载、恶意行为的网站，请不要前来交流。 

https://blog.debuginn.cn/links/

**Comment by owenzjg** [comment]
名称： Owenzjg的小窝
简介： 生活与编程
网址： https://blog.owenzjg.com
头像： https://blog.owenzjg.com/static/img/f4c3416ebf602d043d68ec7cca395fba.72wo_com_gsnndiY4.webp

感谢

**Reply by debuginn** [reply]
你好 请添加本站友情链接再申请哈～

**Comment by gorpeln** [comment]
你好，已加贵站友联（https://gorpeln.top/links），可以交换一下不：

网站名称　　gorpeln 的个人博客 - gorpeln
网站地址　　https://gorpeln.top
网站图标　　https://gorpeln.top/favicon.ico
网站描述　　趁年轻，做自己想做的！

**Reply by debuginn** [reply]
已经添加了～

**Comment by HBUzxl** [comment]
你好，友链中的zz1984是一个色情网站

**Reply by debuginn** [reply]
好滴 感谢反馈哦 我已经下掉了

**Comment by SinzMise** [comment]
名称： 中弦星空魔法屋
简介： 种下一颗有故事的种子，让它带着魔法和奇迹生根发芽
网址： https://blog.sinzmise.top/
头像： https://blog.sinzmise.top/img/512.png

**Comment by SinzMise** [comment]
原 中弦星空魔法屋 已经失效，新的站点信息：
网站名称：汐塔魔法屋
网站链接：https://blog.sinzmise.top/
网站头像：https://blog.sinzmise.top/images/icon.png
网站简介：种下一颗有故事的种子，让它带着魔法和奇迹生根发芽

已添加贵站链接

**Comment by SinzMise** [comment]
```
- title: 汐塔魔法屋
  image: https://blog.storical.space/images/icon.png
  website: https://blog.storical.space/
  description: 种下一颗有故事的种子，让它带着魔法和奇迹生根发芽
```

**Reply by debuginn** [reply]
已经添加，欢迎访问哦～

**Comment by wangxin87532** [comment]
站点名称：王鑫的小站
站点头像：https://pic.wxweb.xyz/pic/main.jpg
站点网址：https://wxweb.xyz
站点简介：活着就是为了折腾！

**Reply by debuginn** [reply]
时间太短哦 小友 再沉淀沉淀吧～ 加油

**Comment by debuginn** [comment]
## 申请友链

> 评论区留言申请～

```yaml
- title: Debug客栈
  image: https://webp.debuginn.com/202302182238057.JPG
  website: https://blog.debuginn.com
  description: 永远相信美好的事情即将发生～
```

## 友情提示

1. 如果您想交换链接，请按上述格式留言。(仅限个人非商业性博客网站)；
2. 网站出现故障、停止维护和不当内容可能会被取消链接！
3. 那些不尊重他人劳动成果、无源转载、恶意行为的网站，请不要前来交流。

**Comment by Vinnie-shen** [comment]
- title: 码农观测站
  image: https://codeobservatory.cn/img/butterfly-icon.webp
  website: https://codeobservatory.cn
  description: 技术分享，编程学习

**Reply by debuginn** [reply]
已经添加，欢迎访问哦～

**Comment by oldte** [comment]
老板，预留个位。  
```
  - title: 老T博客
    description: 法律、科技和生活
    website: https://lawtee.com/
    image: https://lawtee.com/links/avatar.jpg
```

**Reply by debuginn** [reply]
已经添加 1h内生效哈

**Comment by SinzMise** [comment]
名称：绘星里（或者 汐娜物语 ）
描述：一起来绘制属于自己的星星！
地址：https://blog.storia.ren/
头像：https://blog.storia.ren/images/icon.png

（贵站之前删除了本站的友链，当时本站友链还叫“汐塔魔法屋”，后面因space续费续不起才换成现在的fun，到现在才说明，非常抱歉！）

**Reply by debuginn** [reply]
已经恢复了 建议直接买 com 一步到位

**Reply by SinzMise** [reply]
主要com我也不太续得起就是啦😂
而且之前换成fun的时候想让贵站修改本站的友链
但因那个时候贵站的评论区没了，加上我还得春季高考导致没时间，所以我才没去反映的😂

**Reply by debuginn** [reply]
加油 祝你考个好成绩

**Comment by lzh1230** [comment]
博客组织申请友链
网站名称： 博客集市
网站地址： https://cnb.cool/Blog_Bazaar/
网站介绍： 来这里摆摊，让更多人认识你！
网站图标：https://s41.ax1x.com/2026/05/23/pm9EXnS.png

**Comment by ZLVansiit** [comment]
申请友链
title： Z.L Vansiit's blog
website： https://vansiit.cc/
description： 技术博客 | 生活随笔 | 唠嗑扯淡
image：https://vansiit.cc/img/logo.svg

**Reply by debuginn** [reply]
已经添加好友 欢迎多多访问哦～

**Comment by kanghaov** [comment]
Hello, you missed the old friend link~ https://nemo.cool

**Reply by debuginn** [reply]
Nemo 友情链接已经恢复
